cmd/mayactl/app/command/commands.go

```diff
@@ -25,12 +25,12 @@ func NewMayaCommand() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "maya",
```

Comment:
Should these files be under any version ?

Reply:
probably not.. but the type these commands use to interact with mapiserver etc., should be versioned.